recipe/meta.yaml

```diff
@@ -1,25 +1,24 @@
 
 {% set name = "STIR" %}
 {% set version = "5.0.2" %}
-{% set treehash = "rel_5.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/UCL/{{ name }}.git
-  git_rev: {{ treehash }}
+  url: https://github.com/UCL/STIR/archive/refs/tags/rel_{{ version }}.tar.gz
+  sha256: 67c434c78606b2823a47c22f1b02844a594aecccf842c2259c0781a19c9ad8de
```

Comment:
> I know that it is recommended to use url over git_url, although I really don't understand why. It is much more convenient to not have to compute the sha256 so would prefer to keep the use git.

The bot does most of this for you.

In my mind, it really helps ensure that we capture repeatability.

Reply:
How would the bot do this? I create a new STIR version, create a PR here for a new URL. If I don't update the `sha256`, everything will stop.

So, I had to download the package myself, compute the sha256, then update the recipe.

As I want to move this towards complete automation, and don't have the time/energy to figure out how to change the sha256 automatically, I'm afraid I'd rather revert this. (Unless there's a great trick you can point me to!)

Reply:
You create the new STIR tag.

Wait 2-3 hours, the bot will find the new version

The boy will download.

The bot will compute the Sha 

The bot will make the PR

The bot will ask you to review

You can even add an automerge feature so that it also clicks merge 

See for example

https://github.com/conda-forge/zaber-motion-feedstock/pull/89

It did most of the busy work. I removed some patches that were upstreamed.

Reply:
The bot will not issue PR if you give it a git url. So you have to do more work if you revert.

Reply:
sounds fantastic. let's give it a go